app/views/content_items/hmrc_manual.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="govuk-width-container">
+  <div id="manuals-frontend" class="manuals-frontend-body">
```

Comment:
We can remove the parent `<div class="govuk-width-container">` here since we already have the same class on the main wrapper element `<div id="wrapper" class="direction-ltr govuk-width-container">` and the effect is to have double spacing.

<table role="table">
<thead>
<tr>
<th>Live</th>
<th>Dev</th>
</tr>
</thead>
<tbody>
<tr>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/168244255-780f5c1e-f65e-4f57-94c8-d2c36297fb2f.png"><img src="https://user-images.githubusercontent.com/87758239/168244255-780f5c1e-f65e-4f57-94c8-d2c36297fb2f.png" width="360" style="max-width: 100%;"></a></td>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/168244627-03917120-7ffd-4ce4-b034-cebecb46d55b.png"><img src="https://user-images.githubusercontent.com/87758239/168244627-03917120-7ffd-4ce4-b034-cebecb46d55b.png" width="360" style="max-width: 100%;"></a></td>
</tr>
</tbody>
</table>

Reply:
Nice find! I didn't notice the duplicate `govuk-width-container` as I didn't realise this class was injected into the HTML as a result of the Slimmer middleware. Have learnt more about our templating works though with a little quest bouncing from Slimmer to Static to Govuk Publishing Components 😄